### PR TITLE
Fix path to License file

### PR DESCRIPTION
**Pull request by cezarpopa**
Hi,

I hope all is well. I noticed that the license file path was incorrect, so I corrected the path to the LICENSE file in the README.md to ensure it links properly.

Thanks,
Cezar

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Read the documentation at: https://symfony.com/bundles/TailwindBundle/current/in
 
 Is this package useful? We're *thrilled* 😍!
 
-A lot of time & effort from the Symfonycasts team & the Symfony community
+A lot of time and effort from the Symfonycasts team & the Symfony community
```
Comment (bocharsky-bw) [inline README.md:17]: Is there any strong reason for changing & -> and here? I mean, & is the correct shortened version of and anyway, moreover later on this line we still have &. Any comments on this change?

**Review by bocharsky-bw — COMMENTED** [review]
Good catch!

Just a little question about the unrelated change

File: README.md (hunk 1 above)
Comment (cezarpopa) [reply README.md:17]: Ohh, my bad. That shouldn't have been there.

I will remove that in a bit.

Comment (cezarpopa) [reply README.md]: Sorted. Thank you!

**Review by bocharsky-bw — APPROVED** [review]
Beautiful, thank you for this fix!

**Pull request merged**